radixdlt/babylon-wallet-ios
# ABW-2673 Transaction Review: Pool Contribution/Redemption

Jira ticket: [ABW-2673](https://radixdlt.atlassian.net/browse/ABW-2673)

## Description
Adds special handling for contributing to or redeeming from pools

## How to test

Connect wallet and contribute to/redeem from a pool: a custom transaction review view should appear

## Screenshot


## PR submission checklist


[ABW-2673]: https://radixdlt.atlassian.net/browse/ABW-2673?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
